astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -959,6 +959,35 @@ def test_read_big_table(tmpdir):
     assert len(t) == NB_ROWS
 
 
+@pytest.mark.skipif(not os.getenv('TEST_READ_HUGE_FILE'),
```

Comment:
@drdavella , I wonder if it is worth the effort adding a marker for "big data" (like "remote data") for such tests. For instance, https://github.com/spacetelescope/ci_watson/blob/50f527b7ca1682f4befcb63db466cb822deddd84/ci_watson/plugin.py#L43-L44

Reply:
It could be, but where would it go? Here or in `pytest-astropy`? If `ci-watson` already provides it, maybe we should just use that, but it would introduce a new test dependency.

By my count there are only two tests that use this environment variable, and it's doubtful whether either of them ever actually run since this environment variable doesn't appear to be set in any of our CI configs.

Reply:
I wouldn't recommend introducing `ci-watson` as a dependency *here* as that one is very specific to STScI usage. OK, let's table this for now. If Astropy adds more "big data" tests in the future, we'll revisit. Thanks!

Reply:
The purpose seems slightly different from what I understand from your `bigdata` marker. Here a big file is generated, and stored temporarily, which takes time and can only be done on machines with enough memory (~10gb), so not on CI.

Reply:
Question. Should a test run involving running the `TEST_READ_HUGE_FILE` files part of the release procedure? If yes, it would be really nice to have an easy way to run just these tests but not the rest (though I suppose the extra ~+10 min shouldn't make a big difference).

If this has to be part of the procedure, please add it to the release docs. I have to admit I don't always run the full test suite on my laptop before a release, but rather heavily rely on CI, but making sure these tests pass shortly before a release is cut is rather important.

Reply:
@saimn , right, I don't mean using `bigdata` marker as is. I meant in general, as whether we should be using a marker instead of relying on environment variables.

@bsipocz , perhaps we should move this discussion to a new issue?